app/views/kaminari/_prev_page.html.haml

```diff
@@ -0,0 +1,9 @@
+-#  Link to the "Previous" page
```

Comment:
Comment should have a space after the `#`<br>7 consecutive comments can be merged into one